src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -72,11 +74,11 @@ public function getBlockPrefix()
     /**
      * Returns a normalized array of timezone choices.
      */
-    private static function getTimezones(int $regions): array
+    private static function getTimezones(int $regions, int $perCountry = null): array
     {
         $timezones = array();
 
-        foreach (\DateTimeZone::listIdentifiers($regions) as $timezone) {
+        foreach (\DateTimeZone::listIdentifiers($regions, $perCountry) as $timezone) {
```

Comment:
Hm, this is not how listIdentifiers works; you'd pass `regions=PER_COUNTRY, country=US`

https://3v4l.org/SMYkN

Reply:
Ow, OK, ill change the test